csm/core/controllers/s3/accounts.py

```diff
@@ -99,6 +99,7 @@ async def delete(self):
         """Calling Stats Get Method"""
         Log.debug(f"Handling s3 accounts delete request."
                   f" user_id: {self.request.session.credentials.user_id}")
+        self._s3_session = self.request.session.credentials
```

Comment:
i can see the same code is working with SMS1 release. What is the difference ?

Reply:
@kupranay In the latest commit, added _s3_session to its correct place as suggested by @avrnv